novobi_vi_library/controllers/controllers.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.http as http
+
+from odoo.http import request
+from odoo.tools.misc import get_lang
+
+
+class BookController(http.Controller):
+
+    @http.route('/library/book/<string:isbn>', type='http', auth='none', methods=['PUT'], csrf=False)
+    def accept_recurrence(self, isbn, **kwargs):
+        book = request.env['library.book'].sudo().search([
```

Comment:
You should add an option: limit=1 to improve performance
```suggestion
        book = request.env['library.book'].sudo().search([('isbn', '=', ISBN)], limit=1)
```